tests/orquestra/integrations/cirq/simulator/qsimsimulator_test.py

```diff
@@ -29,21 +23,22 @@ def wf_simulator():
 
 
 @pytest.mark.skipif(
-    sys.version_info > (3, 9),
+    sys.version_info > (3, 10),
     reason="QsimSimulator does not work on Python 3.10 or above",
 )
-class TestQsimSimulator(QuantumSimulatorTests):
+# class TestQsimSimulator(QuantumSimulatorTests):
+class TestQsimSimulator:
```

Comment:
We need to add contract testing to this class. Also, the comment is probably not necessary anymore :slightly_smiling_face: